xtask/src/main.rs

```diff
@@ -0,0 +1,107 @@
+//! Custom build system for the Yorick meta-tracer.
```

Comment:
I think something a bit like this probably needs to be in a top-level README: for example, without that no-one will know they need to use something other than `cargo build`. It probably needs to be a cut-down, or at least slightly different, version of this text?

Reply:
Arguably this description (minus implementation details) belongs in ykdocs. How about that?